kategory-optics/src/test/kotlin/kategory/optics/TraversalTest.kt

```diff
@@ -0,0 +1,74 @@
+package kategory.optics
+
+import io.kotlintest.KTestJUnitRunner
+import io.kotlintest.properties.Gen
+import io.kotlintest.properties.forAll
+import kategory.Eq
+import kategory.ListKWHK
+import kategory.TraversalLaws
+import kategory.UnitSpec
+import kategory.genFunctionAToB
+import kategory.genTuple
+import kategory.k
+import kategory.none
+import kategory.some
+import kategory.toT
+import org.junit.runner.RunWith
+import kategory.optics.PTraversal.Companion.fromTraversable
+
+@RunWith(KTestJUnitRunner::class)
+class TraversalTest : UnitSpec() {
+
+    init {
+
+        testLaws(
+                TraversalLaws.laws(
+                        traversal = fromTraversable(),
+                        aGen = Gen.create { Gen.list(Gen.int()).generate().k() },
+                        bGen = Gen.int(),
+                        funcGen = genFunctionAToB(Gen.int()),
+                        EQA = Eq.any(),
+                        EQB = Eq.any()
+                ) + TraversalLaws.laws(
```

Comment:
For now we're separating multiple XXXLaws.laws calls. It'll be a good idea to do a pass to make them like they are in this class and remove potential duplicates. For now I'd keep them separated.